packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -96,13 +96,15 @@ class RefreshIndicator extends StatefulWidget {
     Key key,
     @required this.child,
     this.displacement = 40.0,
+    this.edgeDistance = 0.0,
     @required this.onRefresh,
     this.color,
     this.backgroundColor,
     this.notificationPredicate = defaultScrollNotificationPredicate,
     this.semanticsLabel,
     this.semanticsValue,
   }) : assert(child != null),
+       assert(edgeDistance != null),
```

Comment:
As edgeDistance has a default value of 0.0, would it still be necessary to check for null in this place?

Reply:
Yup, indeed. Submitting a patch.